diff --git a/dist/chunk-D4MB64LD.mjs b/dist/chunk-D4MB64LD.mjs
index 7aa8e3479e66bffa9b55e86c4be265f3c8b2f405..24a02ff1de5bf7bad800daab2581cfce39b6f40d 100644
--- a/dist/chunk-D4MB64LD.mjs
+++ b/dist/chunk-D4MB64LD.mjs
@@ -1,3 +1,5 @@
+import {createRequire as __createRequire} from 'module';
+var require=__createRequire(import.meta.url);
 var __create = Object.create;
 var __defProp = Object.defineProperty;
 var __defProps = Object.defineProperties;
@@ -622,7 +624,7 @@ var logSymbols = isUnicodeSupported() ? main : fallback;
 var log_symbols_default = logSymbols;
 
 // src/release-type.ts
-import { ReleaseType } from "semver";
+// import { ReleaseType } from "semver";
 var prereleaseTypes = ["premajor", "preminor", "prepatch", "prerelease"];
 var releaseTypes = prereleaseTypes.concat(["major", "minor", "patch"]);
 function isPrerelease(value) {
@@ -1297,7 +1299,7 @@ export {
   __toESM,
   require_picocolors,
   log_symbols_default,
-  ReleaseType,
+  // ReleaseType,
   isReleaseType,
   ProgressEvent,
   NpmScript,
diff --git a/dist/index.mjs b/dist/index.mjs
index 444b2f08de65e8d4baa22bc62a4ecc67bc17a0a7..0125275db0aa40433f6eaae4ee5bacf92f8da72c 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1,7 +1,7 @@
 import {
   NpmScript,
   ProgressEvent,
-  ReleaseType,
+  // ReleaseType,
   bumpConfigDefaults,
   defineConfig,
   loadBumpConfig,
@@ -14,7 +14,7 @@ var src_default = versionBump;
 export {
   NpmScript,
   ProgressEvent,
-  ReleaseType,
+  // ReleaseType,
   bumpConfigDefaults,
   src_default as default,
   defineConfig,
